src/js/rx/actions/prescriptions.js

```diff
@@ -1,4 +1,25 @@
-export function loadData() {
+export function loadData(id) {
+  if (id) {
+    // TODO: Use id param instead of test id
+    // when API is able to retrieve any individual Rx.
+    const testId = 1435525;
+    return dispatch => {
+      fetch(`/api/prescriptions/${testId}`)
+      .then(res => res.json())
+      .then(
+        data => dispatch({ type: 'LOAD_PRESCRIPTION_SUCCESS', data }),
+        err => dispatch({ type: 'LOAD_PRESCRIPTION_FAILURE', err })
+      );
+
+      fetch(`/api/prescriptions/${testId}/trackings`)
+      .then(res => res.json())
+      .then(
+        data => dispatch({ type: 'LOAD_RX_TRACKINGS_SUCCESS', data }),
+        err => dispatch({ type: 'LOAD_RX_TRACKINGS_FAILURE', err })
+      );
```

Comment:
Ah, this was an intermediate change that I didn't mean to push up to this branch. But still something that needs to be discussed. I was also considering having the second fetch nested within the resolution of the first one.